lang_qc/endpoints/pacbio_well.py

```diff
@@ -78,17 +78,26 @@
             "description": "Invalid query parameter value"
         }
     },
-    response_model=List[PacBioWell],
+    response_model=PacBioPagedWells
 )
 def get_wells_filtered_by_status(
     qc_status: QcFlowStatusEnum = QcFlowStatusEnum.INBOX,
-    weeks: PositiveInt = 1,
+    page_size: PositiveInt,
+    page_number:PositiveInt,
     qcdb_session: Session = Depends(get_qc_db),
     mlwh_session: Session = Depends(get_mlwh_db),
 ):
 
-    wells, states = grab_wells_with_status(qc_status, qcdb_session, mlwh_session, weeks)
-    return pack_wells_and_states(wells, states)
+    pager = PagedRequest(page_size = page_size, page_number=page_number)
+    wells, states = grab_wells_with_status(qc_status, qcdb_session, mlwh_session)
+    pbwells = pack_wells_and_states(wells, states)
```

Comment:
I think I would be happier if `pack_wells_and_states` went away and was replaced by something more coherent. I think it's hinding and doing too much that is not immediately obvious

Reply:
Agree, should disappear in the next round of refactoring